routing/src/cpi_process.rs

```diff
@@ -3,17 +3,18 @@
 
 //! Main processing functions of the CPI
 
+use crate::evpn::RmacStore;
 use crate::rib::Vrf;
 use crate::rib::VrfTable;
 use crate::routingdb::RoutingDb;
 use crate::rpc_adapt::is_evpn_route;
 use crate::{evpn::RmacEntry, fib::fibobjects::FibGroup};
```

Comment:
Nit: Can you please clean these up?

```
use crate::evpn::{RmacEntry, RmacStore};
use crate::fib::fibobjects::FibGroup;
use crate::rib::{Vrf, VrfTable};
use crate::routingdb::RoutingDb;
use crate::rpc_adapt::is_evpn_route;
```

Reply:
What's wrong with those ?

Reply:
@Fredi-raspall Nothing's “wrong”, it just looks nicer in my opinion to have the includes grouped by module when possible (`use crate::rib::{Vrf, VrfTable};` instead of two separate lines), and to have them sorted (`crate::{evpn::RmacEntry, fib::fibobjects::FibGroup};` in particular makes includes from `evpn` and `fib` happen at the end of the list; and the objects included are not even from the same module, `evpn` vs. `fib::fibobjects`). Do you see any drawback in re-formatting them?

Reply:
I was just asking. I agree that:
`use crate::rib::{Vrf, VrfTable}`
is good, but I personally dislike and tend to put them in two lines.
`crate::{evpn::RmacEntry, fib::fibobjects::FibGroup}`

Reply:
I want to limit the scope of changes here as much as possible, we can fix at a later time.